RELEASE.md

```diff
@@ -10,8 +10,8 @@ Enter `gem install gem-release` to install on your local machine.
 
 ## Release Process
 
+- Use `gem bump -v patch` to bump the patch release number of the gem
 - Update `CHANGELOG.md` with any changes going into this release
-- Use `gem bump -v minor` to bump the minor release number of the gem
```

Comment:
Curiosity question - why is this line removed?

Reply:
It's still there, just the order of the instructions was wrong.

Reply:
Makes sense, thanks!